crates/broker/src/order_picker.rs

```diff
@@ -520,9 +559,11 @@ where
             return Ok(());
         }
 
-        let order_res = self.db.update_orders_for_pricing(capacity).await?;
+        let new_orders = self.db.update_orders_for_pricing(capacity).await?;
+        let lock_expired_orders =
+            self.db.update_slashed_unexpired_for_pricing(capacity, now_timestamp()).await?;
```

Comment:
Will re-using capacity here double the max capacity that the broker can handle? We probably want to let provers specify a separate configuration for fulfilling lockExpired orders.

Reply:
I merged these to functions into 1 that does a more complex db query and respects the same capacity parameter